client/src/components/Input.js

```diff
@@ -0,0 +1,31 @@
+import styled from '@emotion/styled';
+
+const Input = styled.input`
+  outline: none;
+  cursor: pointer;
+`;
+
+export const LoginInput = styled(Input)`
+  text-align: center;
+  width: 274px;
+  height: 60px;
+  background: white;
+  color: white;
+  border: 1px #707070 solid;
+  border-radius: 30px;
+  box-shadow: 0px 3px 6px grey;
+  font-size: 22px;
+  padding: 0px 0px;
+  outline: none;
+  cursor: pointer;
+  ::-webkit-input-placeholder {
```

Comment:
You don't need to set the vendor prefixes. 
```suggestion
  ::placeholder {
```